examples/rllib/view_models.py

```diff
@@ -60,7 +60,7 @@ def main():
   # Create a new environment to visualise
   env = utils.env_creator(config["env_config"]).get_dmlab2d_env()
 
-  num_bots = config["env_config"]["num_players"]
+  num_bots = len(config["env_config"]["roles"])
```

Comment:
This is not quite right... The default number of players should be inferred from the length og the `default_player_roles` in the config.

Reply:
Interesting! I'm afraid I cannot find the config where the `default_player_roles` shall be in.

Reply:
They are in the substrate configs, e.g.:
https://github.com/deepmind/meltingpot/blob/ad1cead1e77138d1f20545f508d6bbc459746582/meltingpot/python/configs/substrates/allelopathic_harvest__open.py#L81

Reply:
Ah got it, thank you!